cmd/podman/root.go

```diff
@@ -207,7 +207,13 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 	// 2) running as non-root
 	// 3) command doesn't require Parent Namespace
 	_, found := cmd.Annotations[registry.ParentNSRequired]
-	if !registry.IsRemote() && rootless.IsRootless() && !found {
+
+	hasCapSysAdmin, err := rootless.HasCapSysAdmin()
+	if err != nil {
+		return err
+	}
+
+	if !registry.IsRemote() && !hasCapSysAdmin && !found {
```

Comment:
The same change must be made here https://github.com/containers/podman/blob/69085570f7ebbb3768e963e2a6a31d7bb9b4ca16/cmd/podman/common/completion.go#L46